agent/api/eni.go

```diff
@@ -86,7 +94,22 @@ func ENIFromACS(acsenis []*ecsacs.ElasticNetworkInterface) []*ENI {
 		})
 	}
 
-	return enis
+	return enis, nil
+}
+
+func ValidateTaskENI(acsenis []*ecsacs.ElasticNetworkInterface) error {
+	// Only one eni should be associated with the task
+	// Only one ipv4 should be associated with the eni
+	// ONly one ipv6 should be associated with the eni
+	if len(acsenis) != 1 {
+		return errors.Errorf("enifromacs: task contains %d eni,expected one", len(acsenis))
```

Comment:
Can you make this more user friendly? `"eni message validation: more than one ENIs in the message (%d)"`